packages/react-native/tooling/posthog-xcode.sh

```diff
@@ -84,6 +84,19 @@ else
 fi
 set -x -e # re-enable
 
+# files wont exist if skip bundling
+set +x +e
+if [[ "$SKIP_BUNDLING" ]]; then
+  echo "SKIP_BUNDLING enabled; skipping posthog-cli upload calls."
+  exit 0;
+fi
+
+if [[ ! "$FORCE_BUNDLING" ]]; then
+  echo "Skipping bundling in Debug for the Simulator (since the packager bundles for you). skipping posthog-cli upload calls."
+  exit 0;
+fi
+set -x -e
```

Comment:
debug builds via xcode fails now if not skipping since the files wont exist anymore